modules/portmaster/src/i18n/helptexts.yaml

```diff
@@ -41,95 +40,92 @@ uiMode:
 pilot-widget-NetworkRating:
   title: Network Rating
   content: |
-    Some features and settings of the Portmaster behave differently, depending on the environment.
+    Control your privacy even when connecting to new networks.
```

Comment:
The network does not have to be new - situations can change while being connected.

Reply:
I know that this is not technically correct since there are other potential scenarios. However, we are explaining this feature step by step. And for the first step teaching users to _always_ check this when connecting to a new network is a good thing. Later down the road (@Danger) they learn that this can even be done when being in the same network.

Reply:
I see. Ok. Let's watch out for feedback regarding this.